components/Trade/Advanced/RightPanel/index.tsx

```diff
@@ -197,8 +197,13 @@ const StyledSlideSelect = styled(SlideSelect)`
         color: #fff;
     }
 
-    @media (max-height: 850px) {
+    @media (max-height: 1080px) {
         display: flex;
+
+        #orderbook-dropdown {
+            display: flex;
+            align-items: center;
+        }
```

Comment:
Would try and avoid doing something like this in conjunction with the ID. The reason being that this is using the id to add css when there is already css for this Option below. Have committed an alternative